test/known_issues/test-vm-inherited_properties.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+// Ref: https://github.com/nodejs/node/issues/5350
+
+require('../common');
+const vm = require('vm');
+const assert = require('assert');
+
+const base = {
+  prop_base: 1,
+};
+
+const sandbox = Object.create(base, {
+  prop_sandbox: {value: 3}
+});
+
+const context = vm.createContext(sandbox);
+
+const result = vm.runInContext("this.hasOwnProperty('prop_base');", context);
```

Comment:
Is this the proper way to write it?
```'this.hasOwnProperty("propBase");'```
I thought that double quotes are used in JS to avoid escaping: http://standardjs.com/rules.html
Could you clarify this for me, please? Thanks!